docs/authentication.md

```
@@ -0,0 +1,141 @@
+
+この分離により、signup後に離脱したユーザーのゴミデータがDBに残らない。
+
+## ユーザー状態
+
+| 状態 | Auth0 | DB | 説明 |
+|------|-------|-----|------|
+| 未認証 | × | × | ログインしていない |
+| 認証済み・未登録 | ○ | × | Auth0でsignup済み、アプリ未登録 |
+| 登録済み | ○ | ○ | アプリのユーザーとして有効 |
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)